ogr/services/pagure/service.py

```diff
@@ -332,6 +332,8 @@ def project_create(
             parameters["description"] = repo
         if namespace:
             parameters["namespace"] = namespace
+        else:
+            namespace = self.user.get_username()
```

Comment:
@mfocko in some cases, changing the code rather than the typing just seemed like the more correct solution. This is one of the places which causes the tests to fail. If `namespace` is not provided, it seems to assume the current authenticated user and fills it in automatically. However, `PagureProject` constructor requires namespace. If I hadn't added this extra call (which obviously creates the need for test data regeneration), the only solution would be to make `namespace` inside `PagureProject` `Optional` which does not seem right.